try:
    print(fddsss)
except NameError:
    print("whoops")
else:
    print("try worked")
finally:
    print("always happens")